language-examples/python/vectara-rest/rest_upload_file.py

```diff
@@ -54,4 +54,10 @@ def upload_file(customer_id: int, corpus_id: int, idx_address: str, jwt_token: s
                        response.reason,
                        response.text)
         return response, False
-    return response, True
\ No newline at end of file
+
+    message = response.json()["response"]
+    if message["status"]:
```

Comment:
Shouldn't you also check that status is not OK?

Reply:
Going back to the point Laurence made previously, it feels like this should also be set to OK. Consistency across all our APIs can only help our users.

Reply:
Done.